x-pack/plugins/apm/server/routes/service_map/get_service_map_service_node_info.ts

```diff
@@ -122,12 +143,16 @@ async function getTransactionStats({
   filter,
   minutes,
   searchAggregatedTransactions,
-}: TaskParameters): Promise<{
-  avgTransactionDuration: number | null;
-  avgRequestsPerMinute: number | null;
-}> {
+  start,
+  end,
+  intervalString,
+}: TaskParameters): Promise<NodeStats['transactionStats']> {
   const { apmEventClient } = setup;
 
+  const field = getTransactionDurationFieldForTransactions(
```

Comment:
Better to be more specific
```suggestion
  const durationField = getTransactionDurationFieldForTransactions(
```

Reply:
Btw. can you rename `getTransactionDurationFieldForTransactions` to `getDurationFieldForTransactions`. The repetition of "transaction" looks terrible once you've seen it 🙈 

```suggestion
  const durationField = getDurationFieldForTransactions(
```